website/public/locale/en/translation.yaml

```diff
@@ -265,7 +265,7 @@ community:
       message: CNCF hosted OpenEBS community events
 
   sponsor:
-    mainSponsor: Started and majority of its development sponsored by
+    mainSponsor: Majority of Development sponsored by
```

Comment:
```suggestion
    mainSponsor: Majority of development sponsored by
```

Reply:
This is in accordance with the text change request mentioned in the Email.